vms/platformvm/txs/executor/tx_mempool_verifier.go

```diff
@@ -136,7 +135,7 @@ func NextBlockTime(state state.Chain, clk *mockable.Clock) (time.Time, bool, err
 
 	nextStakerChangeTime, err := GetNextStakerChangeTime(state)
 	if err != nil {
-		return time.Time{}, false, fmt.Errorf("could not calculate next staker change time: %w", err)
+		return time.Time{}, false, err
```

Comment:
This should be wrapped by the caller since it's the only error that can be returned in this func